addons/survey/views/survey_templates.xml

```diff
@@ -402,6 +403,62 @@
         </div>
     </template>
 
+    <template id="question_scale" name="Question: scale">
+        <!-- A question already answered that the user resets is marked as skipped and not deleted, -->
+        <!-- so we use the flag skipped to restore or not the answer state. -->
+        <t t-set="answer_line" t-value="answer_lines and not answer_lines[0].skipped and answer_lines[0]"/>
+        <div class="o_survey_answer_wrapper o_survey_form_choice"
+             t-att-data-name="question.id"
+             t-att-data-is-skipped-question="is_skipped_question or None"
+             data-question-type="scale">
+            <div role="radiogroup" class="btn-group d-flex mb-2" t-att-aria-label="">
+                <t t-foreach="range(question.scale_min, question.scale_max + 1)" t-as="value">
+                    <t t-set="item_idx" t-value="value_index"/>
+                    <t t-set="answer_selected" t-value="answer_line and answer_line.value_integer == value"/>
+                    <label t-attf-class="o_survey_choice_btn flex-shrink-1 rounded text-nowrap #{'me-2' if not value_last else ''} #{'o_survey_selected' if answer_selected else ''}"
+                           t-attf-for="{{ question.id }}_{{ value }}">
+                        <div class="d-none d-md-block">
+                            <div class="d-flex justify-content-between align-items-center align-content-center my-3">
+                                <div/>
+                                <div class="d-flex align-items-center align-content-center">
+                                    <div class="fs-5 fw-bold" t-out="value"/>
+                                </div>
+                                <div class="d-flex align-items-center align-content-center">
+                                    <i class="fa fa-check-circle me-1"/>
+                                    <i class="fa fa-circle-thin me-1"/>
+                                </div>
+                            </div>
+                        </div>
```

Comment:
Lots of nested div ... isn't it possible to simplify the dom a bit ? Like a d-flex holding another div just for t-out value, ... seems we generally achieve simpler dom.